src/dotnet/APIView/.editorconfig

```diff
@@ -0,0 +1,4 @@
+﻿[*.cs]
+
+# RS1024: Compare symbols correctly
+dotnet_diagnostic.RS1024.severity = none
```

Comment:
Is there any reason not to use `#pragma warning disable` for this?

Reply:
This check is broken. Roslyn marked the bugs as fixed (https://github.com/dotnet/roslyn-analyzers/pull/2337), but for some reason it still shows an error for comparing two interface instances using `object.Equal`.